app/Jobs/DeleteGenre.php

```diff
@@ -63,14 +63,17 @@ public function handle()
 
         // This is done instead of a single UPDATE query in order to
         // generate revision logs for the change.
-        $this->genreToDelete->tracks()->each(function ($track) {
-            /** @var Track $track */
-
-            $track->genre_id = $this->destinationGenre->id;
-            $track->save();
-            $track->updateTags();
-        });
+        $tracks = Track::whereGenreId($this->genreToDelete->id)->get();
 
         $this->genreToDelete->delete();
+        $chunks = $tracks->chunk(200);
```

Comment:
For some reason, chunking wouldn't work before. As in, the code inside the function that should run on every chunk just didn't run. Having the function return a collection of collections is the only way I could get this to work. It works fine in my dev environment. Thoughts @Feld0 ?